integration/integration_test.go

```diff
@@ -1950,9 +1973,18 @@ func testHA(t *testing.T, suite *integrationTestSuite) {
 	require.NoError(t, b.Start())
 	require.NoError(t, a.Start())
 
-	nodePorts := helpers.NewPortSlice(3)
-	sshPort, proxyWebPort, proxySSHPort := nodePorts[0], nodePorts[1], nodePorts[2]
-	require.NoError(t, a.StartNodeAndProxy("cluster-a-node", sshPort, proxyWebPort, proxySSHPort))
+	// The Listener FDs injected into SiteA will be closed when SiteA restarts
+	// later in in the test, rendering them all invalid. This will make SiteA
+	// fail when it attempts to start back up again. We can't just inject a
+	// totally new listener config into SiteA when it restarts, or SiteB won't
+	// be able to  find it.
+	//
+	// The least bad option is to duplicate all of SiteA's Listener FDs and
+	// inject those duplicates prior to restarting the SiteA cluster.
+	aFdCache, err := a.Process.ExportFileDescriptors()
+	require.NoError(t, err)
+
+	sshPort, _, _ := a.StartNodeAndProxy(t, "cluster-a-node")
```

Comment:
Is the error ignored here?

Reply:
Amy errors are caught inside `StartNodeAndProxy()` and result in an immediately failed test.